package_control/dependency.py

```diff
@@ -101,13 +101,13 @@ def install(dest_root, src_dir, name, version, description, url, plat_specific):
 
     di_record_path = os.path.join(dist_info_path, 'RECORD')
     # Create an empty file so it shows up in its own file list
-    with open(di_record_path, 'wb') as f:
-        f.write(b'')
+    with open(di_record_path, 'w', encoding='utf-8') as fobj:
```

Comment:
This could have remained as using binary mode. In fact, the following line can probably be removed/replaced with `pass` as well.

Reply:
Sure. I am not even sure, whether `write` is even required with regards to other usages which just do `open().close()` with same result.

Reply:
Seeing how often empty files are created, a utility function to do just that seems reasonable, but of course not required.

Reply:
Not sure whether it's really worth it.